cli/lsp/diagnostics.rs

```diff
@@ -351,16 +453,18 @@ impl DiagnosticsServer {
                   }
                   let mark =
                     performance.mark("update_diagnostics_deps", None::<()>);
-                  let diagnostics = generate_deno_diagnostics(
-                    &snapshot,
-                    &config,
-                    token.clone(),
-                  )
-                  .await;
+                  let diagnostics = spawn_blocking({
+                    let token = token.clone();
+                    move || generate_deno_diagnostics(&snapshot, &config, token)
+                  })
+                  .await
+                  .unwrap();
```

Comment:
Also gets the deno and lint diagnostics in blocking tasks, which should speed things up.